### PR TITLE
Prepare for release 1.4.10

### PR DESCRIPTION
**Pull request by damonbarry**

### CODE REVIEW

**Comment by damonbarry** [comment]
@varunpuranik, @nyanzebra, @bishal41 Please review the commit descriptions in the changelog.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,52 @@
...
+* azureiotedge-agent
+* azureiotedge-hub
+* azureiotedge-simulated-temperature-sensor
+* azureiotedge-diagnostics
+
+## Edge Agent
+### Bug fixes
+* Update versions of .NET dependencies to patch security vulnerabilities ( [82ca5e8](https://github.com/Azure/iotedge/commit/82ca5e8eb082a3d5ebf8276fc0109f923c022ad2) )
+* Update version of Azure IoT SDK to fix a memory leak ( [d98f43c](https://github.com/Azure/iotedge/commit/d98f43c90d456f51903ee2fa3f7266083086d739) )
+* Optionally detect and remove orphaned module identities on startup ( [3bac802](https://github.com/Azure/iotedge/commit/3bac80274305c7f2d4af92c161bead2c486d6820) )
```
Comment (nyanzebra) [inline CHANGELOG.md:19]: What is the verbage we normally use for when EdgeAgent brings modules up? startup or deployment?

**Review by nyanzebra — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (damonbarry) [reply CHANGELOG.md:19]: A deployment happens whenever edge agent receives a twin update that changes the config. I don't think there's a precise tech term for when edge agent is first starting up (including reconciling the config).

Comment (nyanzebra) [reply CHANGELOG.md:19]: So, the check happens on receiving twin, so maybe deployment will be more accurate term to use here then.

Comment (damonbarry) [reply CHANGELOG.md]: Done

**Review by bishal41 — APPROVED** [review]
LTGM, @damonbarry

**Pull request merged**